examples/harvest-vm/README.md

```diff
@@ -0,0 +1,135 @@
+
+# Running Placement Policy Scheduler Plugins in Harvest VM cluster
+
+
+## AKS demo
+
+#### 1. Create [AKS](https://docs.microsoft.com/en-us/azure/aks/) cluster
+
+```sh
+# Create a resource group in East US
+az group create --name harvestvm --location eastus
+
+# Create a basic single-node AKS cluster
+az aks create \
+    --resource-group harvestvm \
+    --name harvestaks \
+    --vm-set-type VirtualMachineScaleSets \
+    --node-vm-size Harvest_E2s_v3
+    --node-count 3 \
+    --generate-ssh-keys \
+    --load-balancer-sku standard
+```
+
+When the cluster is ready, use the az aks get-credentials command to get the cluster credentials for use with kubectl:
+
+```
+az aks get-credentials --resource-group harvestvm --name harvestaks
+```
+
+Add a second node pool that runs 3 nodes:
+
+```
+az aks nodepool add \
+    --resource-group harvestvm \
+    --cluster-name harvestaks \
+    --name normalvms \
+    --node-count 3
+```
+
+#### 2. Deploy placement-policy-scheduler-plugins as a secondary scheduler
+
+The container images for the scheduler plugin is available in the github container registry.
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/Azure/placement-policy-scheduler-plugins/main/deploy/kube-scheduler-configuration.yml
+```
+
+<details>
+<summary>Result</summary>
+
+```
+customresourcedefinition.apiextensions.k8s.io/placementpolicies.placement-policy.scheduling.x-k8s.io created
+configmap/pp-scheduler-config created
+clusterrole.rbac.authorization.k8s.io/pp-plugins-scheduler created
+clusterrolebinding.rbac.authorization.k8s.io/pp-plugins-scheduler created
+rolebinding.rbac.authorization.k8s.io/pp-plugins-scheduler-as-kube-scheduler created
+clusterrolebinding.rbac.authorization.k8s.io/pp-plugins-scheduler-as-kube-scheduler created
+serviceaccount/pp-plugins-scheduler created
+deployment.apps/pp-plugins-scheduler created
+
+```
+</details><br/>
+
+#### 3. Deploy a `PlacementPolicy` CRD
+
+```yaml
+kind: PlacementPolicy
+metadata:
+  name: harvest-stric-must
+spec:
+  weight: 100
+  enforcementMode: Strict
+  podSelector:
+    matchLabels:
+      app: nginx
+  nodeSelector:
+    matchLabels:
+      # instansce type can be one of the following( Harvest_E2s_v3, Harvest_E4s_v3, Harvest_E8s_v3)
```

Comment:
```suggestion
      # instance type can be one of the following( Harvest_E2s_v3, Harvest_E4s_v3, Harvest_E8s_v3)
```

Reply:
Can we also add a step for how to get this node label for the existing nodes?

Reply:
Please let me know what you think [step 3](https://github.com/Azure/placement-policy-scheduler-plugins/pull/55/files/493684b947b9964145ca36748ef44efb80bdd84a#diff-5112de02c93dd5f001ff64f0e5b79191c18bd697fffed12283fc55e8a1f70271R64)